src/main/resources/com/adobe/epubcheck/schema/20/rng/opf20.rng

```diff
@@ -168,12 +166,9 @@ Authors:
       <data type="ID"/>
     </attribute>
     </optional>
-    <ref name="OPF20.optional-xsi-type"/>
+
     <ref name="OPF20.optional-scheme-attribute"/>
-    <!-- <ref name="DC.metadata-common-content"/>
-    MG 20111024, see epubcheck Issue 114
-    -->
-    <ref name="DC.metadata-required-content"/>    
+    <ref name="DC.metadata-common-content"/>
```

Comment:
oh, and this may also interfer with #114 

cannot say if the changes from #114 are reflected in the latest schema version from the specs...